packages/itwinui-react/src/core/ExpandableBlock/ExpandableBlock.tsx

```diff
@@ -119,6 +130,7 @@ export const ExpandableBlock = React.forwardRef((props, ref) => {
           'iui-expanded': expanded,
           'iui-small': size === 'small',
           'iui-borderless': styleType === 'borderless',
+          'iui-disabled': disabled,
```

Comment:
`iui-disabled` is not present in the CSS so it does nothing and can be removed.

Reply:
fixed